### DISCUSSION TITLE
群发消息 收不到 没报错

### DISCUSSION THREAD

**Discussion by JacianZhu** [simple-robot/Discussions#158] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

群发消息 收不到 没报错

 for (BotManager<?> botManager : application.getBotManagers()) {
                if (botManager instanceof MiraiBotManager) {
                    Items<Group> group = botManager.all().get(0).getGroups();
                    List<Group> list = group.collectToList();
                    if (!list.isEmpty()) {
                        for (Group item : list) {
                            String message = groupMessageService.getMessage(10);
                            item.sendBlocking(message);
                        }
                    }
                }
            }

一开始还好的了 跑了大概30分钟 也不报错 就是收不到消息了

**Comment by ForteScarlet** [comment]
尝试发送私聊、发送简单文字（例如 `1`、`2` 之类的）。发不出消息但是没有报错大概率是风控的原因。
其他参考：
- https://github.com/mamoe/mirai/issues/2688
